docs/notebooks/feedback_prod.ipynb

```diff
@@ -0,0 +1,333 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<!-- docusaurus_head_meta::start\n",
+    "---\n",
+    "title: Log Feedback from Production\n",
+    "---\n",
+    "docusaurus_head_meta::end -->\n",
+    "\n",
+    "<img src=\"http://wandb.me/logo-im-png\" width=\"400\" alt=\"Weights & Biases\" />\n",
+    "<!--- @wandbcode{feedback-colab} -->"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "vscode": {
+     "languageId": "plaintext"
+    }
+   },
+   "source": [
+    "\n",
+    "It is often hard to automatically evaluate a generated LLM response so, depending on your risk tolerance, you can gather direct user feedback to find areas to improve.\n",
+    "\n",
+    "In this tutorial, we'll use a custom RAG chatbot as an example app with which the users can interact and which allows us to collect user feedback.\n",
+    "We'll use Streamlit to build the interface and we'll capture the LLM interactions and feedback in Weave."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "vscode": {
+     "languageId": "plaintext"
+    }
+   },
+   "source": [
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install weave openai streamlit"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, create a file called `secrets.toml` and add an OpenAI key so it works with `[st.secrets](https://docs.streamlit.io/develop/api-reference/connections/st.secrets)`. You can [sign up](https://platform.openai.com/signup) on the OpenAI platform to get your own API key. "
```

Comment:
Storing secret keys in a plaintext file seems like encouraging bad practices - does getting OPENAI_API_KEY from the environment not work? If we are telling people to put it in a file, we should probably at least direct them to `~/.streamlit/` which is less likely to get checked in and leak the key.

Reply:
this is how they're recommending it [here](https://docs.streamlit.io/deploy/streamlit-community-cloud/deploy-your-app/secrets-management), along with telling users to gitignore the file